EndlessSky.xcodeproj/project.pbxproj

```diff
@@ -1055,7 +1096,7 @@
 				"$(TEMP_ROOT)/sdl2.dmg",
 			);
 			outputPaths = (
-				"$(TEMP_ROOT)/SDL2.framework",
+				"$(PROJECT_DIR)/build/SDL2.framework",
```

Comment:
For "Run Script" build phases, [Xcode will skip the phase](https://developer.apple.com/documentation/Xcode/improving-the-speed-of-incremental-builds?changes=_8) if a) both input and output files are specified, b) the input files haven't changed, and c) the output files exist. I realized the fetch_sdl2_framework.sh script wasn't placing SDL2.framework in the temp directory, causing the build phase to run on each build. Since sdl2.dmg is no longer needed once the framework is copied out of it, it's left in the temp dir, e.g. `~/Library/Developer/Xcode/DerivedData/EndlessSky-gmlixeyoidmypubngcofffsyugqu/Build/Intermediates.noindex/sdl2.dmg`. (I also realize I'd omitted an input file from the libmad target's script build phase — I'll push a fix shortly.)